src/main/java/entity/Recipe.java

```diff
@@ -30,44 +41,66 @@ public void printRecipe() {
         System.out.println(this.diets);
         this.nutrition.printNutrition();
     }
+
+    /**
+     * Retrieves name of Recipe.
+     * @return returns name of Recipe.
+     */
+
     public String getName() {
         return name;
     }
+
+    /**
+     * Retrieves Recipe's nutritional facts.
+     * @return nutritional facts of recipe.
+     */
     public Nutrition getNutrition() {
         return nutrition;
     }
 
+    /**
+     * Retrieves dietary restrictions the recipe adheres to.
+     * @return dietary restrictions the recipe adheres to through a map and a boolean.
+     */
+
     public Map<String, Boolean> getDiets() {
         return diets;
     }
 
+    /**
+     * Retrieves cooking duration of recipe.
+     * @return cooking duration of recipe in an int that represents minutes.
+     */
+
     public int getCookingTime() {
         return cookingTime;
     }
 
+    /**
+     * Retrieves instructions of recipe.
+     * @return instructions of recipe.
+     */
+
     public String getInstructions() {
         return instructions;
     }
 
+    /**
+     * Retrieves total calories of the recipe.
+     * @return total calories of recipe.
+     */
+
     public int getTotalCalories() {
         return nutrition.getCalories();
     }
 
+    /**
+     * Retrieves ingredients of recipe.
+     * @return a list of ingredients of the recipe.
+     */
     public List<String> getIngredients() {
         return ingredients;
     }
 
-    public Recipe getAllDetails() {
-        Map<String, Object> details = new HashMap<>();
-        details.put("name", name);
-        details.put("ingredients", ingredients);
-        details.put("instructions", instructions);
-        details.put("cookingTime", cookingTime);
-        details.put("diets", diets);
-        details.put("nutrition", nutrition);
-        return (Recipe) details;
-        // this is new! and will def cause a merge conflict
-    }
-
-
-}
\ No newline at end of file
+}
```

Comment:
To adhere to the Checkstyle, you should add one more line at the end. Other than that, perfect! I'll review my code again since there could be some missing javadocs.